pylint/checkers/refactoring.py

```diff
@@ -52,7 +52,13 @@ class RefactoringChecker(checkers.BaseTokenChecker):
                   'suggest a potential error. This is taken in account only for '
                   'a handful of name binding operations, such as for iteration, '
                   'with statement assignment and exception handler assignment.'
-                 )
+                 ),
+        'R1705': ('Superfluous return before else',
```

Comment:
We can use the same messages as ESLint, for consistency: "Disallow return before else" with the symbolic name no-else-return.